### PR TITLE
Enable mutant in --since mode globally

### LINKED ISSUE #109: Reenable mutation testing once global style fixes are done.

### PR DESCRIPTION
**Pull request by mbj**
[fix #109]

### CODE REVIEW

File: shared/Rakefile [hunk 1]
```diff
@@ -50,6 +48,7 @@ namespace :metrics do
       --use rspec
       --require ./spec/dummy/config/environment.rb
       --since HEAD~1
+      --jobs 1
```
Comment (mbj) [inline shared/Rakefile]: @dkubb Once we ported our advanced mutation testing compatible database cleaner we can go up with the jobs.

**Comment by mbj** [comment]
@dkubb This is ready for review, but GH does not seem to realize the green status (also its properly rebased). I'll force an SHA change to nuke caches.

**Comment by dkubb** [comment]
This change looks good to me. I will merge when green.

**Comment by dkubb** [comment]
One thing I noticed is that there's no entries in buildkite to run mutant against the code.

**Comment by mbj** [comment]
> One thing I noticed is that there's no entries in buildkite to run mutant against the code.

good catch, fixed.

**Comment by mbj** [comment]
@dkubb Ready for the next pass.

**Comment by dkubb** [comment]
This change looks good to me. I will merge when green.

**Pull request merged**